acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/privacy/contract/CallPrivateSmartContractFunction.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl.privacy.contract;
+
+import tech.pegasys.pantheon.tests.acceptance.dsl.transaction.NodeRequests;
+import tech.pegasys.pantheon.tests.acceptance.dsl.transaction.Transaction;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.List;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.pantheon.Pantheon;
+import org.web3j.tx.LegacyPrivateTransactionManager;
+import org.web3j.tx.PrivateTransactionManager;
+import org.web3j.tx.gas.PantheonPrivacyGasProvider;
+import org.web3j.utils.Base64String;
+
+public class CallPrivateSmartContractFunction implements Transaction<String> {
+
+  private static PantheonPrivacyGasProvider GAS_POROVIDER =
+      new PantheonPrivacyGasProvider(BigInteger.valueOf(1000));
+  private final String contractAddress;
```

Comment:
Of "Address" type?

Reply:
same as above. Essentially I would be converting the web3j to a Pantheon type and then back to a web3j types for assertion